dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -171,3 +171,6 @@ throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an illegal state excepti
 
 getRealPathReturnsQualifiedPath.name=Always return a non-null path from getRealPath
 getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false.
+
+stopAppStartUponListenerException.name=Stop the web application when context listener fails to start due to an unhandled exception.
+stopAppStartUponListenerException.desc=Some web applications depend on context listeners to do the necessary setup before the web application is started. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners.
```

Comment:
Change,
"Some web applications depend on context listeners to do the necessary setup before the web application started. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners.
8 dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml "
to,
"Some web applications depend on context listeners for setup before the web application starts. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners.
8 dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml "